Dockerfile

```diff
@@ -24,6 +24,7 @@ ENV NOTIFY_BASE_API_URL=https://api.notification.canada.ca
 ENV NOTIFY_API_KEY=""
 ENV NOTIFY_REPORT_A_PROBLEM_EMAIL=""
 ENV NOTIFY_REPORT_A_PROBLEM_TEMPLATE_ID=""
+ENV ADOBE_ENABLED = true
```

Comment:
Looks like we're not using this anymore, so we can remove this.

Reply:
Removed this.